docs/getting-started/start-app.md

```diff
@@ -45,5 +45,86 @@ instance/tab of your shell.
 Current gotchas: potential environment issues with external services need to be
 worked out.
 
+# Starting the application (advanced)
+
+To have additional control on the local environment you might prefer using an
+advanced setup to start the application.
+
+The prerequisite is to install [Overmind][overmind], a process manager which
+takes advantage of [tmux][tmux].
+
+This will allow you to launch everything in the same terminal, navigate the logs
```

Comment:
```suggestion
This will allow you to launch all your app's processes in the same terminal, navigate the logs
```
Not sure if this is actually correct, please use a better word if I am wrong in my suggestion here 😛 Just figured we could be a little more specific than "everything"

Reply:
definitely correct!